File: java/Samples/BiometricStandards/CS/CbeffRecordSampleCS/AddRecordOptionsForm.cs
using System;
using Neurotec.Biometrics.Standards;

namespace Neurotec.Samples
{
	public partial class AddRecordOptionsForm : Neurotec.Samples.CbeffRecordOptionsForm
	{
		#region public properties

		public BdifStandard Standard
		{
			get
			{
				return (BdifStandard)cbStandard.SelectedItem;
			}
		}

		#endregion

		#region public constructor

		public AddRecordOptionsForm()
		{
			InitializeComponent();

			cbStandard.Items.Add(BdifStandard.Ansi);
			cbStandard.Items.Add(BdifStandard.Iso);
			cbStandard.SelectedIndex = 0;
		}

		#endregion
	}
}

File: java/Samples/BiometricStandards/VB/CbeffRecordSampleVB/AddRecordOptionsForm.vb
Imports Neurotec.Biometrics.Standards

Partial Public Class AddRecordOptionsForm
	Inherits CbeffRecordOptionsForm
#Region "public properties"

	Public ReadOnly Property Standard() As BdifStandard
		Get
			Return CType(cbStandard.SelectedItem, BdifStandard)
		End Get
	End Property

#End Region

#Region "public constructor"

	Public Sub New()
		InitializeComponent()

		cbStandard.Items.Add(BdifStandard.Ansi)
		cbStandard.Items.Add(BdifStandard.Iso)
		cbStandard.SelectedIndex = 0
	End Sub

#End Region
End Class
